nova/nova/views.py

```diff
@@ -61,6 +59,63 @@ def user_res(request, pk):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+@api_view(['GET', 'POST'])
+def user_followings_coll(request, pk):
+    try:
+        user = User.objects.get(pk=pk)
+    except User.DoesNotExist:
+        raise NotFound()
+
+    if request.method == 'GET':
+        followings = user.followings
+        serializer = UserSerializer(followings, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    elif request.method == 'POST':
+        if request.user.pk != pk:
+            raise PermissionDenied()
+        if 'following_pk' not in request.data:
+            raise ValidationError()
+        try:
+            following = User.objects.get(pk=request.data['following_pk'])
+        except User.DoesNotExist:
+            raise NotFound()
+
+        request.user.followings.add(following)
+        serializer = UserSerializer(following)
+
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+
+@api_view(['DELETE'])
+def user_following_res(request, user_pk, following_pk):
+    try:
+        user = User.objects.get(pk=user_pk)
+        following = User.objects.get(pk=following_pk)
+    except User.DoesNotExist:
+        raise NotFound()
+
+    if request.method == 'DELETE':
+        if request.user.pk != user_pk:
```

Comment:
This is great! I'd forgotten to add a similar check to the update user endpoint (you should be able to update a user iff you are that user or you are an admin.) Maybe we can create an issue about that.

Reply:
I added this check to the PUT request in your migration PR. If more change is needed like this, I agree that we can open an issue for that.

Reply:
Oh hadn't seen that. Thanks!